workspaces/analytics/tests/DeclaringEvents.test.ts

```diff
@@ -0,0 +1,7 @@
+const { ApiCreated } = require('../src/events/onboarding');
+
+test('Can creeate an event type', () => {
```

Comment:
The opportunity to fix spelling errors is now, otherwise you'll also have to edit the snapshot file